cinema/models.py

```diff
@@ -27,6 +27,10 @@ class Actor(models.Model):
     first_name = models.CharField(max_length=255)
     last_name = models.CharField(max_length=255)
 
+    @property
+    def full_name(self):
+        return f"{self.first_name} {self.last_name}"
+
     def __str__(self):
         return self.first_name + " " + self.last_name
```

Comment:
1) add a type annotation
2) These two methods do the same thing. Maybe it's better to use one of them in the other?

Reply:
Sorry. I do not agree with point number 2. The two methods are the same, but if I leave, for example, only the str method and change it in the future, then I will lose the functionality of the full_name field.  I want full_name to be independent, so I created this property.

Reply:
Sorry. I realized. I'll just do `return full_name()` in `str`
Thanks

Reply:
btw, I partially agree about independence. But if you leave these methods independent, then they should at least be written in the same way inside (adhere to the same style)